package.json

```diff
@@ -3,6 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@enabledao/enable-contracts": "^0.1.0",
```

Comment:
Doesn't the `npm` registry reject packages below `1.0.0`? I remember some funky behavior

https://docs.npmjs.com/about-semantic-versioning

Reply:
Interesting, I didn't know about that. It seemed to publish and download just fine.

Reply:
Hm yeah makes sense. Let's keep it below 1.0 then!